README.md

```diff
@@ -30,8 +79,8 @@ Measure the power consumption of the following scenarios. For each scenario, cre
 
 To eliminate any bias in the measurements, use an external LED with a known current value limiting resistor on a GPIO pin. Don't use the built in LED, which requires the wireless module to be enabled. If you want a serial connection, you need to use the UART pins rather than USB and access it with a serial converter.
 
-Blink an LED, using sleep_ms to delay iterations.
-Blink an LED, using FreeRTOS with a thread.
-Run a busy loop to get 100% CPU utilization
-Place #pragma GCC optimize ("O0") at the top of the file to disable compiler optimizations.
-Run a tight loop like the one below. Multiplication typically requires large amounts of silicon, and adders with a full carry chain causes lots of activity. I don't know if these will be the worst instructions for energy usage, but this should be a decent place to start.
+1) Blink an LED, using sleep_ms to delay iterations.
+2) Blink an LED, using FreeRTOS with a thread.
+3) Run a busy loop to get 100% CPU utilization
```

Comment:
What you have as 3), 4), and 5) are all one activity. 4) and 5) are notes suggesting how to do this. There are also 3 more scenarios (really 4 or 5 if you count using different frequencies):

1. Set up an interrupt to toggle a GPIO in response to an external signal (like in lab 5).
    1. In the main loop to invoke the wait for interrupt instruction.
    1. Drive the interrupt pin with the signal generator.
    1. Try a couple of frequencies between 1KHz and 1 MHz.
1. Run the sleep demo. https://github.com/raspberrypi/pico-playground/tree/master/sleep/hello_sleep/hello_sleep_alarm.c
    1. More details are in section 2.11.5.1 of the RP2040 datasheet.
    1. Read through the demo source and the library code to make sure that the code matches the setup description in the datasheet
    1. https://github.com/raspberrypi/pico-extras/blob/master/src/rp2_common/pico_sleep/sleep.c
1. Run the dormant demo. https://github.com/raspberrypi/pico-playground/tree/master/sleep/hello_dormant/hello_dormant_gpio.c
    1. More details are in section 2.11.5.2 of the RP2040 datasheet.